src/wiki/development/build-instructions.md

```diff
@@ -166,36 +166,56 @@ We recommend using a build workflow based on MSYS2, as it's the easiest way to g
+   - If you want to build using Qt 5, then remove the `-DLauncher_QT_VERSION_MAJOR=6` parameter
+   - If you want to use [ccache](#ccache) to speed up recompilations, add the parameter `-DCMAKE_CXX_COMPILER_LAUNCHER=ccache`
 
 3. Now you need to run the build itself: Run `cmake --build build`.
 4. Now, wait for it to compile. This could take some time, so hopefully it compiles properly.
 5. Run the command `cmake --install build`, and it should install Prism Launcher to whatever the `-DCMAKE_INSTALL_PREFIX` was.
 6. If you don't want Prism Launcher to store its data in `%APPDATA%`, run `cmake --install build --component portable` after the install process
-7. When building on Qt 5, whenever compiling, the OpenSSL DLLs aren't put into the directory to where Prism Launcher installs which are necessary in that case, meaning that you cannot log in. The best way to fix this, is just to do `cp /(msystem))/bin/libcrypto-1_1.dll /(msystem))/bin/libssl-1_1.dll install`. This should copy the required OpenSSL DLLs to log in. When building on Qt 6 this is not necessary because it can use schannel, the Windows tls library.
+7. When building on Qt 5, whenever compiling, the OpenSSL DLLs aren't put into the directory to where Prism Launcher installs which are necessary in that case, meaning that you cannot log in. The best way to fix this, is just to do `cp /(msystem)/bin/libcrypto-1_1.dll /(msystem)/bin/libssl-1_1.dll install`. This should copy the required OpenSSL DLLs to log in. When building on Qt 6 this is not necessary because it can use schannel, the Windows tls library.
 
-   - Replace *(msystem)* with the msystem you're using (e.g. on *MSYS2 MinGW CLANG x86* clang32). On 64 bit msystems like *MSYS2 MinGW CLANG x64*, you have to add a -x64 to the dlls.
+   - Replace `(msystem)` with the msystem you're using (e.g. clang64). On 64-bit msystems, like *MSYS2 CLANG64*, you have to add `-x64` to the dlls.
+
+### Using Qt Creator (optional)
+
+1. Install Qt Creator within MSYS2 using `pacboy -S qt-creator:p`
+
+   - NOTE: If you install Qt Creator within Windows itself, Qt Creator will not be able to find anything installed in MSYS2.
+
+2. (Optional) Create a shortcut to `C:\msys64\(msystem).exe qtcreator`
+
+   - Replace `(msystem)` with the msystem you're using (e.g. clang64).
+
+3. Open Qt Creator with the `qtcreator` command in MSYS2.
+4. Choose `File->Open File or Project`.
+5. Navigate to the Launcher source folder you cloned and choose `CMakeLists.txt`.
+6. When prompted to configure the project, scroll past the **many** Desktop Qt options without changing anything and click "Configure Project" at the bottom right.
+7. Cross your fingers, and press the "Run" button (bottom left of Qt Creator).
+
+   - If the project builds successfully it will run and the Launcher window will pop up.
+
+**If this doesn't work for you, please let us know on our Discord sever, or Matrix Space.**
```

Comment:
We don't have a matrix space yet, I think you can remove this

Reply:
*yet*

Reply:
Also, the Matrix space is still mentioned in the Linux instructions.